### PR TITLE
Fixes broken import

### PR DESCRIPTION
**Pull request by alextp**

### CODE REVIEW

**Comment by alextp** [comment]
@MarkDaoust can you take a look? The colab is currently broken and this fixes it

File: site/en/guide/autograph.ipynb [hunk 1]
```diff
@@ -139,7 +139,7 @@
         "from __future__ import division, print_function, absolute_import\n",
         "\n",
         "import tensorflow as tf\n",
-        "import tensorflow.keras.layers as layers\n",
+        "layes = tf.keras.layers\n",
```
Comment (brilee) [inline site/en/guide/autograph.ipynb]: typo - layers

**Comment by alextp** [comment]
Fixed the typo

**Comment by mdanatg** [comment]
Looks like that import line needs to be fixed too. I couldn't edit that branch but you can find it in https://github.com/tensorflow/docs/pull/52

**Pull request merged**